lib/ansible/cli/doc.py

```diff
@@ -52,12 +52,14 @@ def __init__(self, args):
     def parse(self):
 
         self.parser = CLI.base_parser(
-            usage='usage: %prog [-l|-s] [options] [-t <plugin type] [plugin]',
+            usage='usage: %prog [-l|-f|-s] [options] [-t <plugin type] [plugin]',
```

Comment:
dont use -f as it is already use for forks in other CLIs

Reply:
suggest alternative and I will change it.

Reply:
-F